kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
Or would ezplatformsearch work fine with https://github.com/ezsystems/ezpublish-legacy/blob/master/cronjobs/indexcontent.php ? Which handles the kernel pending index actions? I'm also fine with that, but then we need to expand doc in ezplatformsearch for that right?